### PR TITLE
Pass $action through the alternateAbsoluteLink() method

### PR DESCRIPTION
**Pull request by threesquared**
This allows you to use `->AbsoluteLink('action')`

### CODE REVIEW

File: code/extensions/SiteTreeSubsites.php [hunk 1]
```diff
@@ -208,10 +208,10 @@ static function contentcontrollerInit($controller) {
 		if($subsite && $subsite->Theme) SSViewer::set_theme(Subsite::currentSubsite()->Theme);
 	}
 
-	function alternateAbsoluteLink() {
+	function alternateAbsoluteLink($action) {
```
Comment (dhensby) [inline code/extensions/SiteTreeSubsites.php]: this needs to be an optional parameter, I'd have thought?

**Comment by dhensby** [comment]
I think this would be useful against 3 / 3.2 and not just master.

File: code/extensions/SiteTreeSubsites.php (hunk 1 above)
Comment (threesquared) [inline code/extensions/SiteTreeSubsites.php]: An argument is passed through from the call in SiteTree (https://github.com/silverstripe/silverstripe-cms/blob/3.2/code/model/SiteTree.php#L477) but you are right, better to be safe.

**Comment by tractorcow** [comment]
Please rebase on master.

**Comment by wernerkrauss** [comment]
I think this be closed, as it's fixed in v4 now. Thanks for your effort @threesquared

**Pull request closed without merging**